src/Const/README.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Const"
+description: "Const Monoid"
+layout: "guide"
+weight: 80
+---
+
+```haskell
+Const c _
+```
+`Const` a unary function which creates a simple Product type that evaluates to
+`c` for all inputs ignoring its right side. 
+`Const` is a `chainable` that will take any value `c` and regardless of the 
+function called on it will return a new `Const` with the value `c`.
+
+### Contrived Examples
+```javascript
+import Const from 'crocks/Const'
+
+Const('Hello World')
+//=> Const 'Hello World'
+
+const days = [ 'Today', 'Tomorrow', 'Yesterday' ]
+
+days.map(Const).reduce((acc, c) => acc.concat(c))
+//=> Const 'Today'
+
+Const(100)
+  .concat(Const(10))
+//=> Min 100
+
+Const('Hello')
+  .map(x => x + x)
+//=> Const 'Hello'
+```
+
+### Real-world example
+```javascript
```

Comment:
You might consider describing in english what this data flow does. i.e. `Update field allows us to store an updated value in the second slot of a Pair, and resetField allows us to reset that second slot to the constant value in the first`